src/main/java/com/zaxxer/nuprocess/linux/LinuxProcess.java

```diff
@@ -183,41 +183,6 @@ private void closePipes()
       LibC.close(stderrWidow);
    }
 
-   @Override
-   protected boolean checkLaunch()
```

Comment:
This didn't show as unused because it was overriding a base class method, but it was never actually called